go/extras.rst

```diff
@@ -1,95 +0,0 @@
-Extra rules
```

Comment:
Please don't delete the file, instead leave a stub file that says "This file has moved, please look at it here"